### ISSUE TITLE
Multiple GPUs for training

### ISSUE THREAD

**Issue by DarylWM** [rishizek/tensorflow-deeplab-v3#2]
Thanks for sharing this code @rishizek.

Has anyone managed to train this on more than one GPU?

**Comment by John1231983** [comment]
Hi, I tried it but it was not successful. The reason is that training with batch norm in multiple GPU. It is not supported in tensorflow yet

**Comment by DarylWM** [comment]
Ok, thanks.
